tests/test_downloader.py

```diff
@@ -434,11 +434,11 @@ class BaseSpider(Spider):
 
```

Comment:
Some tests need to be updated since it's now getting other warnings since we've used `default_registry.get_rules()`.